examples/js/loaders/GLTF2Loader.js

```diff
@@ -2111,6 +2112,93 @@ THREE.GLTF2Loader = ( function () {
 						meshNode = new THREE.Mesh( geometry, material );
 						meshNode.castShadow = true;
 
+						if ( primitive.targets !== undefined ) {
+
+							geometry.morphTargets = [];
```

Comment:
With #11285 is merged, creating `geometry.morphTargets = [];` should not be necessary anymore. Instead, you can set the name of the morph target directly to a `morphAttribute`, see https://github.com/mrdoob/three.js/issues/11277#issuecomment-299420508.